test/production/next/jest/index.test.ts

```diff
@@ -23,6 +27,7 @@ describe('next/jest', () => {
           export default function Page() { 
             return <>
               <Comp />
+              <Image src={img} alt="logo"/>
```

Comment:
Do we need one with `placeholder='blur'` as well and have mock `blurData` included?